docs/docs/ref-01-top-level-api.md

```diff
@@ -90,7 +90,11 @@ Verifies the object is a ReactElement.
 
 ### React.Children
 
-`React.Children` provides utilities for dealing with the `this.props.children` opaque data structure.
+`React.Children` provides utilities for dealing with the `this.props.children` data structure.
+
+> **Note:**
+>
+> In most cases, you want to treat `this.props.children` as an opaque object. In reality, its structure can have a number of possibilities. We call it a "React node", which is analogous to DOM nodes which can be elements, text nodes, or fragments containing multiple other nodes. Each React node can be a React element, a string, a number, null/true/false/undefined (all ignored), or an array or iterable containing other React nodes.
```

Comment:
I would just ditch this note.  I don't think it adds anything useful (beyond what I wrote above), and it opens a whole other can of worms.  For example, you didn't mention `Symbol` - intentional?  You said it could be an iterable containing React nodes... So when I do `<Foo children={new myIterable(...)} />` will `React.children.map(Foo.props.children)` have a single element (the iterable) or will it map over all the values in the iterable (I could make the argument either way, it's really not clear which behavior is correct).